sources/Adapters/picoTracker/gui/picoTrackerEventManager.cpp

```diff
@@ -54,15 +54,20 @@ int picoTrackerEventManager::MainLoop() {
       queue->Empty(); // Avoid duplicates redraw
       redrawing_ = false;
     }
+
+    if (loops == (3 * 1000000)) {
```

Comment:
This is a dirty hack, wouldn't use this TBH. Plus this is super approximate

Reply:
yes good point! I'll change it per the event injection suggestion below.